src/aoscoreconfig.hpp

```diff
@@ -72,4 +84,14 @@
  */
 #define AOS_CONFIG_SERVICEMANAGER_SERVICES_DIR CONFIG_AOS_SERVICES_DIR
 
+/**
+ * Set launcher thread stack size.
+ */
+
+#define AOS_CONFIG_LAUNCHER_THREAD_STACK_SIZE CONFIG_AOS_LAUNCHER_THREAD_STACK_SIZE
+/**
+ * Configures thread stack usage.
```

Comment:
Not a clear description. 
Recommend changing to: PRINT_STACK_USAGE,  better to utilize with #ifdef so 1 is redundant.

Reply:
updated